bot.py

```diff
@@ -44,58 +44,78 @@
 #variables needed for !uwu
 #I mean what else would it be for?
 json_db['uwu_suffixes'] = [
-    ' ( ͡° ᴥ ͡°)',
     ' (´・ω・｀)',
-    ' (ʘᗩʘ\')',
-    ' (இωஇ )',
     ' (๑•́ ₃ •̀๑)',
     ' (• o •)',
     ' (⁎˃ᆺ˂)',
     ' (╯﹏╰）',
     ' (●´ω｀●)',
     ' (◠‿◠✿)',
     ' (✿ ♡‿♡)',
-    ' (❁´◡`❁)',
+    ' (❁´◡\`❁)',
     ' (　\'◟ \')',
-    ' (人◕ω◕)',
     ' (；ω；)',
-    ' (｀へ´)',
-    ' ._.',
+    ' (´･ω･\`)',
+    ' o3o'
     ' :3',
     ' :D',
     ' :P',
-    ' ;-;',
-    ' ;3',
-    ' ;_;',
+    ' ;\_;',
     ' <{^v^}>',
-    ' >_<',
-    ' >_>',
+    ' >\_<',
     ' UwU',
-    ' XDDD',
     ' ^-^',
-    ' ^_^',
-    ' x3',
-    ' x3',
     ' xD',
     ' ÙωÙ',
-    ' ʕʘ‿ʘʔ',
     ' ㅇㅅㅇ',
     ' （＾ｖ＾）'
+    ' \*kisses you and licks your neck\*',
```

Comment:
I... Uh... I think i need a counselor for this one. If you can get @capktkirk to sign off on it, I'll allow it

Reply:
The two below this one as well lol

Reply:
For reference, these suffixes were lifted from the uwu copypasta linked below. I think they fit in pretty well and definitely make for some interesting results, but if you think it's too much for the server I can take them out.

https://genius.com/Reddit-copypastas-uwu-whats-this-annotated

Reply:
They're just hyper sexual in nature. I think for most cases, it's fine, but it might be offputting if a bot starts coming on to a user who's not expecting it.

If you want to get @capktkirk in on this, I'm happy to accept it, otherwise, maybe not?